### ISSUE TITLE
Dockerized Sentora Web Hosting Panel

### ISSUE THREAD

**Issue by prashant2594** [sentora/sentora-installers#130]
Hi all, I've been trying to create a sentora container, but it fails every time, with the below error,
![1](https://user-images.githubusercontent.com/44703864/78120418-8c9f0600-7427-11ea-98ed-7447a65788ec.png)

Can anyone please me out, in creating sentora image, and how to run sentora on docker contianers.

**Comment by bradjtrammell** [comment]
I don't believe you will be able to "dockerize" Sentora because of the requirements it needs to operate. Is there any specific reason it needs to be utilized via docker or is this just a case of "I want to see if it's possible", just curious as this is the first I've seen of someone looking to do this.

**Comment by Dukecitysolutions** [comment]
Looking into it in the future!
